### DISCUSSION TITLE
Jyers v135

### DISCUSSION THREAD

**Discussion by jethroc0457** [Jyers/Marlin#1320] [Q&A] [answered]
I installed this firmware when I go to print  it does the bed leveling then asks me ti install filament then it just pushes heaps of filament out of the nozzel that fast it makes the feeder  slip
How do you fix this issue

**Comment by vali20** [comment]
Do you have filmanet sensor option enabled?
Try again after you disable it.

**Reply by jethroc0457** [reply]
No I have it disabled

**Comment by kted** [comment]
Did you reset to factory defaults after flashing?

**Comment by jethroc0457** [comment, marked as answer]
The problem was something in cura g-coding I unistalled cura and then reinstalled it an now all is good
